packages/v4/patternfly-docs/content/design-guidelines/components/data-list/data-list.md

```diff
@@ -51,14 +54,15 @@ See [when to use compact vs. default spacing](#when-to-use-a-data-list-vs.-a-tab
 ### Data list capabilities
 Every data list can be extended with these functionalities:
 
-* **[Actionable data list](/components/data-list/design-guidelines#actionable-data-list)**
+* **[Selectable data list](/components/data-list/design-guidelines#selectable-data-list)**
```

Comment:
These first 3 quick links don't work ! To make these quick links, you only need the last part with the #.. so it should be 

[selectable data list](#selectable-data-list)

(Check the draggable data list row one as an example)